CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary.
 
- ## Next
+## Next
+
+## 0.14.1 (2023/10/17)
+
+### Bug Fixes
```

Comment:
I think this should be `Security`. That seems to be what we have used in the past for things like this.

Reply:
Oh, TIL! Addressed with https://github.com/hashicorp/boundary/pull/3851/commits/ea53302945207c0f8aa91207fd545c717d388a18